server/src/http/refresher/feature_refresher.rs

```diff
@@ -347,7 +349,7 @@ impl FeatureRefresher {
 
                                     match serde_json::from_str(&event.data) {
                                         Ok(features) => { refresher.handle_client_features_updated(&token, features, None).await; }
-                                        Err(e) => { warn!("Could not parse features response to internal representation: {e:?}");
+                                        Err(e) => { tracing::error!("Could not parse features response to internal representation: {e:?}");
```

Comment:
Ok, this is the relevant change, shouldn't we use the same `match serde_json::from_str` in line 424 and 425 when reading deltas?